lib/p2p/Pool.ts

```diff
@@ -41,7 +42,7 @@ class Pool extends EventEmitter {
     this.hosts = new HostList(new P2PRepository(db));
   }
 
-  get peerCount(): number {
+  public get peerCount(): number {
```

Comment:
Is there any locking mechanism for the data structure to avoid parallel update/deletes?

Reply:
It's a single threaded process, so there won't be race conditions as far as I know.